src/OpenTelemetry.Instrumentation.SqlClient/README.md

```diff
@@ -123,19 +114,23 @@ and `System.Data.SqlClient` are limited:
   query text.
 
 Since `CommandType.Text` might contain sensitive data, all SQL capturing is
-_disabled_ by default to protect against accidentally sending full query text
+_disabled_ by default when using [`Microsoft.Data.SqlClient`](https://www.nuget.org/packages/Microsoft.Data.SqlClient/)
+under .NET Framework to protect against accidentally sending full query text
 to a telemetry backend. If you are only using stored procedures or have no
 sensitive data in your `sqlCommand.CommandText`, you can enable SQL capturing
 using the options like below:
 
 ```csharp
 using var tracerProvider = Sdk.CreateTracerProviderBuilder()
     .AddSqlClientInstrumentation(
-        options => options.SetDbStatement = true)
+        options => options.SetDbStatementForText = true)
     .AddConsoleExporter()
     .Build();
 ```
 
+The `SetDbStatementForText` setting has no effect when using [`Microsoft.Data.SqlClient`](https://www.nuget.org/packages/Microsoft.Data.SqlClient/)
```

Comment:
Add that `RecordException` has no effect when using .NET Fw.

Reply:
It already says available only on .NET core in the readme. And the XML comment says that it has no effect on framework.